plugins/modules/events.py

```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+DOCUMENTATION = r"""
+---
+module: events
+
+short_description: Handle AssistedInstall cluster events
+
+version_added: "1.0.0"
+
+description: Interact with cluster events through the AssistedInstall API
+
+options:
+    action:
+        description: The action to perform.
+        required: false
+        default: "list"
+        type: str
+    cluster_id:
+        description: The cluster ID to perform the action on.
+        type: str
+    limit:
+        description: The maximum number of records/events to retrieve.
+        required: false
+        type: int
+    order:
+        description: Retrieval order for cluster events based on time of events.
+        default: ascending
+        required: false
+        type: str
+        choices: [ ascending, descending ]
+    offset:
+        description: Number of events to skip before events retrieval.
+        required: false
+        type: int
+        default: 0
+    severities:
+        description: Retrieve events mapped to the severity value.
+        required: false
+        type: list
+        elements: str
+        choices: [ info, warning, error, critical ]
+
+author:
+    - Akash Gopalakrishnan (@agopalak)
+"""
+
+EXAMPLES = r"""
+# Use argument
+- name: List cluster events
+  events:
+    action: list
+    cluster_id: "deadmeat-dead-meat-dead-meatdeadmeat"
+    limit: 50
+    order: descending
+    offset: 10
+    severities:
+        - "critical"
+        - "info"
+"""
+
+RETURN = r"""
+cluster_events:
+    description: A list of cluster events
+    type: dict
```

Comment:
@agopala95 maybe this needs to be `type: list` and you want on the next line `elements: str`? See https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#return-block

Reply:
Thanks for the suggestion. Resolved it by adding `elements` to the argument spec as per @tonyskapunk's hint.